Resources/ANOVA/qml/AncovaBayesian.qml

```diff
@@ -206,47 +206,67 @@ Form
 			title: qsTr("Prior")
 			DoubleField { name: "priorFixedEffects";		label: qsTr("r scale fixed effects");	defaultValue: 0.5;	 max: 2; inclusive: "maxOnly"; decimals: 3 }
 			DoubleField { name: "priorRandomEffects";		label: qsTr("r scale random effects");	defaultValue: 1;	 max: 2; inclusive: "maxOnly"; decimals: 3 }
-            DoubleField { name: "priorCovariates";	        label: qsTr("r scale covariates");		defaultValue: 0.354; max: 2; inclusive: "maxOnly"; decimals: 3 }
+			DoubleField { name: "priorCovariates";			label: qsTr("r scale covariates");		defaultValue: 0.354; max: 2; inclusive: "maxOnly"; decimals: 3 }
 		}
 
-        RadioButtonGroup
+		RadioButtonGroup
 		{
 			name: "sampleModeNumAcc"
 			title: qsTr("Numerical Accuracy")
-            RadioButton { value: "auto";	label: qsTr("Auto"); checked: true }
+			RadioButton { value: "auto";	label: qsTr("Auto"); checked: true }
 			RadioButton
 			{
-                value: "manual";	label: qsTr("Manual")
+				value: "manual";	label: qsTr("Manual")
 				IntegerField
 				{
 					name: "fixedNumAcc"
-                    label: qsTr("No. samples")
+					label: qsTr("No. samples")
 					defaultValue: 1e4
 					fieldWidth: 50
-                    min: 100
-                    max: 1e7
+					min: 100
+					max: 1e7
 				}
 			}
 		}
 
-        RadioButtonGroup
+		RadioButtonGroup
 		{
 			name: "sampleModeMCMC"
 			title: qsTr("Posterior Samples")
-            RadioButton { value: "auto";	label: qsTr("Auto"); checked: true }
+			RadioButton { value: "auto";	label: qsTr("Auto"); checked: true }
 			RadioButton
 			{
-                value: "manual";	label: qsTr("Manual")
+				value: "manual";	label: qsTr("Manual")
 				IntegerField
 				{
 					name: "fixedMCMCSamples"
-                    label: qsTr("No. samples")
+					label: qsTr("No. samples")
 					defaultValue: 1e3
 					fieldWidth: 50
-                    min: 100
-                    max: 1e7
+					min: 100
+					max: 1e7
 				}
 			}
 		}
+
```

Comment:
Okay, this works, but ideally, we make a new QML component out of this so that it's truly standardized across all analyses (instead of copy-pasting code).

To add a new qml component I always mimic `BayesFactorType.qml`:

- Add a new QML file in `jasp-desktop/JASP-Desktop/components/JASP/`, e.g., `SetSeed.qml`.
- In this file, put all the stuff you want to reuse.
- Reference the file to `jasp-desktop/JASP-Desktop/components/JASP/qmldir` and `jasp-desktop/JASP-Desktop/qml.qrc`.

Now it should be possible to use `SetSeed{}` in all QML files!

Reply:
Hmm, I tried to do that but I'm still getting this error from Qt:
(the code is in the last commit)

qmlcache_loader.obj:-1: error: LNK2001: Unrecognized external symbol "unsigned char const * const QmlCacheGeneratedCode::_components_JASP_Widgets_SetSeed_qml::qmlData" (?qmlData@_components_JASP_Widgets_SetSeed_qml@QmlCacheGeneratedCode@@3QBEB)

Reply:
Bruno helped to fix the issue (I needed to reset cache of my JASP build)